src/styles/blocks/_product-card.scss

```diff
@@ -0,0 +1,103 @@
+.product-card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+
+  font-family: $font-family-base;
+  font-weight: $fw-normal;
+
+  background-color: $color-white;
+  border: 1px solid $border-color;
+  border-radius: $border-radius;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__content {
+    width: $content-width;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: $fw-medium;
+    line-height: $lh-large;
+    color: $color-main-accent;
+
+    margin-top: 40px;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    line-height: $lh-small;
+    color: $color-secondary;
+
+    margin-top: 4px;
+  }
+
+  &__review-wrapper {
+    @include space-between;
+
+    align-items: center;
+
+    margin-top: 16px;
+  }
+
+  &__review {
+    font-size: 10px;
+    line-height: $lh-small;
+    color: $color-main-accent;
+  }
+
+  &__price-wrapper {
+    @include space-between;
+
+    margin-top: 24px;
+  }
```

Comment:
nest it inside price

Reply:
It can't be in price because price and text must be in wrapper, if I do that it will lose the application of styles due to increased specificity and the readability of the code will deteriorate. Wouldn't it?

Reply:
you're not increasing specificity because you're only nesting not putting it inside:
```
&__price {
    font-size: 16px;
    font-weight: $fw-bold;
    line-height: $lh-large;
    color: $color-main-accent;

    &-wrapper {
      @include space-between;

      margin-top: 24px;
    }
  }
```
![image](https://github.com/mate-academy/layout_product-cards/assets/126968958/ad4b13c1-4b26-4436-a43f-fb2bfc274e56)

Reply:
![image](https://github.com/mate-academy/layout_product-cards/assets/56163379/b50c1188-1432-4b83-b7f4-ab1b48790a65)